projects/canopy/src/lib/accordion/accordion-item/accordion-item.component.scss

```diff
@@ -9,5 +9,6 @@
 
   &--active {
     display: block;
+    padding-top: var(--space-md);
```

Comment:
I would add this to line 3 of this file. Also the same has to be done for the Details component.

Reply:
Done 
<img width="1207" alt="Screenshot 2023-04-27 at 15 59 48" src="https://user-images.githubusercontent.com/8149965/234870592-632a22bc-7dec-4fdc-a39a-5295c74dc032.png">